From 2ce6b5ae1c4f4617b7c4f650c9dd4d27d4662d3c Mon Sep 17 00:00:00 2001
From: Pavithra R <quic_pavir@quicinc.com>
Date: Tue, 12 Sep 2023 17:53:01 +0530
Subject: [PATCH 364/500] arm: Fix compilation issue on 512MB 32 bit debug
 build

Compiler is not able to find the definition for the parameter 'current'
with 32bit build as 'current.h' for 32bit is not included in other
header files. So including the header file to compile 32bit.

Change-Id: I1e52bb48899c070ebac8909c43cdda5ee137a839
Signed-off-by: Pavithra R <quic_pavir@quicinc.com>
---
 arch/arm/include/asm/stacktrace.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/include/asm/stacktrace.h b/arch/arm/include/asm/stacktrace.h
index 360f0d2406bf..89143f1c4461 100644
--- a/arch/arm/include/asm/stacktrace.h
+++ b/arch/arm/include/asm/stacktrace.h
@@ -3,6 +3,7 @@
 #define __ASM_STACKTRACE_H
 
 #include <asm/ptrace.h>
+#include <asm/current.h>
 #include <linux/llist.h>
 
 struct stackframe {
-- 
2.34.1

